### PR TITLE
Sprint tweak

### PR DESCRIPTION
**Pull request by jakkeez**

### CODE REVIEW

**Review by LeoWikstrom — DISMISSED** [review]

**Review by LeoWikstrom — DISMISSED** [review]

File: StortSpelProjekt/Game/src/main.cpp [hunk 1]
```diff
@@ -171,7 +171,7 @@ Scene* GameScene(SceneManager* sm)
     melc = entity->AddComponent<component::MeleeComponent>();
```
Comment (kattalainen) [inline StortSpelProjekt/Game/src/main.cpp:171]: on row 172, could you please add this:     melc->SetDamage(10);
this somehow got lost in merges, thank you!

File: StortSpelProjekt/Game/src/Gamefiles/Components/PlayerInputComponent.cpp [hunk 2]
```diff
@@ -256,12 +256,12 @@ void component::PlayerInputComponent::move(MovementInput* evnt)
 			if (vel == double3({ 0.0, 0.0, 0.0 }))
 			{
 				forward.normalize();
-				vel = { forward.x * BASE_VEL * DASH_MOD, forward.y, forward.z * BASE_VEL * DASH_MOD };
+				vel = { forward.x * m_MovementSpeed * DASH_MOD, forward.y, forward.z * m_MovementSpeed * DASH_MOD };
```
Comment (kattalainen) [inline StortSpelProjekt/Game/src/Gamefiles/Components/PlayerInputComponent.cpp:259]: plz tone down the dash a bit, you can now glitch through the thick outer walls and into the ground in the "inside area"

**Review by kattalainen — DISMISSED** [review]

File: StortSpelProjekt/Vendor/config.txt [hunk 3]
```diff
@@ -1,11 +1,11 @@
-f_volume 10.0
+f_volume 3.0
 f_backgroundVolume 0.2
 f_updateRate 30
 f_networkUpdateRate 15
-i_resolutionWidth 1365
-i_resolutionHeight 768
-i_windowWidth 1366
-i_windowHeight 768
+i_resolutionWidth 1921
```
Comment (fjz345) [inline StortSpelProjekt/Vendor/config.txt]: 1920

**Review by fjz345 — CHANGES_REQUESTED** [review]
Change resolution. Then also tweeks that should be made:
1. Fullscreen
2. Camera distance to player
3. Mouse sensitivity?

**Review by fjz345 — APPROVED** [review]

**Review by kattalainen — APPROVED** [review]

**Pull request merged**